src/app/components/substeps-details/substeps-details.component.html

```diff
@@ -1,2 +1,32 @@
-<!-- TODO fill out component template -->
-{{substep|json}}
+<mat-card-content>
+    <dl class="substep-details">
+        <div>
+            <dt>Cleared</dt>
+            <dd>
+                {{substep?.cleared}}
+            </dd>
+        </div>
+
+        <div>
+            <dt>Substep ID</dt>
+            <dd>{{substep?.substepId}}</dd>
+        </div>
+        <div>
+            <dt>Substep Nr</dt>
+            <dd>{{substep?.substepNr}}</dd>
+        </div>
+        <div>
+            <dt>Processor Location</dt>
+            <dd>{{substep?.href}}</dd>
+        </div>
+        <!-- TODO: add ui href too? -->
```

Comment:
Remove todo. UI href is only used to display the microfrontend of an active substep => cleared substeps should not show a micro frontend any more